ui/src/app/pages/admin/user/admin-user-egress-bypass.spec.tsx

```diff
@@ -5,27 +5,113 @@ import * as React from 'react';
 import { UserAdminApi } from 'generated/fetch';
 
 import { render, screen } from '@testing-library/react';
-import { registerApiClient } from 'app/services/swagger-fetch-clients';
+import userEvent from '@testing-library/user-event';
```

Comment:
test changes here were not ultimately needed for this PR, but I think they're a nice addition